public/client.js

```diff
@@ -7,7 +7,7 @@ COBI.devkit.overrideThumbControllerMapping.write(true);
 COBI.app.clockVisible.write(false);
 
 // Initialize slider if in experience or overview mode (therefore not in edit menu)
-if (COBI.parameters.state() == COBI.state.experience || COBI.parameters.state() == COBI.state.overview) {
+if (COBI.parameters.context() != COBI.context.onRideSettings && COBI.parameters.context() != COBI.context.offRideSettings) {
```

Comment:
@dfarache this is changing the logic from a positive OR to a negative AND. I dont think it is a problem but I cannot be sure of it. I would prefer if you keep it the old way for consistency but it is not a hard requirement

Reply:
@carocad I understand what you say. I wanted to do it like that but I had a problem translating _state_ into _context_:

Would you say this is true:
`state.experience OR state.overview == context.offRide  OR context.onRide`

Reply:
yes that sounds about right